### ISSUE TITLE
Errors when I try to update drupal 8.3.5 to 8.4.0

### ISSUE THREAD

**Issue by FlorianLeMenn** [drush-ops/drush#3060]
When I try to update drupal 8.3.5 to 8.4.0 I have errors:

```
PHP Fatal error:  Class 'Drupal\locale\TranslationString' not found in /home/webmaster/www-data-dev/drupal8/core/modules/locale/src/StringDatabaseStorage.php on line 77

Fatal error: Class 'Drupal\locale\TranslationString' not found in /home/webmaster/www-data-dev/drupal8/core/modules/locale/src/StringDatabaseStorage.php on line 77
PHP Fatal error:  Class 'Drupal\locale\TranslationString' not found in /home/webmaster/www-data-dev/drupal8/core/modules/locale/src/StringDatabaseStorage.php on line 77

Fatal error: Class 'Drupal\locale\TranslationString' not found in /home/webmaster/www-data-dev/drupal8/core/modules/locale/src/StringDatabaseStorage.php on line 77
```

Drush Version   :  8.1.13
![kittty_2017-10-12_14-31-20](https://user-images.githubusercontent.com/4291551/31495732-01bee734-af59-11e7-8b07-baadb074f6bc.png)

**Comment by greg-1-anderson** [comment]
Not sure what happened here, but these errors appear to be unrelated to Drush. Maybe try removing your composer.lock and vendor directory and do a fresh `composer install`, and see if that fixes things up.

**Comment by calbasi** [comment]
Hi,
I've found this error when upgrading from drupal 8.3.7 to 8.4.5 in a site with 2 enabled languages different from (default) English (Catalan and Spanish). I've solved it just enabling English, upgrading after it (drush up --security-only).

Once upgrade is done, I've deleted English language again.

Ps: after first attempt to upgrade, the site was broken. But a new backup folder drupal-8.4.5 folder was created. Copy its content to your root web folder to restore your previous site ;-)
